### PR TITLE
[HM] MPL Link 2

### PR DESCRIPTION
**Pull request by FZill**
Continuation of the Linkage of HM to MPL.
Solid density and biot coefficient were adapted.
Added a namespace alias for the MaterialPropertyLib, since previously it resulted in bulky statements when fetching the template values.

Also, I simplified the porosity specification in some benchmark files.

I used the same script as last time: 
[ConvertToMPL.txt](https://github.com/ufz/ogs/files/4121439/ConvertToMPL.txt)

### CODE REVIEW

**Comment by FZill** [comment]
Just fixed some misaligned indent in one benchmark file

File: ProcessLib/HydroMechanics/HydroMechanicsProcessData.h [hunk 1]
```diff
@@ -56,10 +56,6 @@ struct HydroMechanicsProcessData
     ParameterLib::Parameter<double> const& intrinsic_permeability;
     /// Fluid's density. A scalar quantity, ParameterLib::Parameter<double>.
     ParameterLib::Parameter<double> const& fluid_density;
```
Comment (wenqing) [inline ProcessLib/HydroMechanics/HydroMechanicsProcessData.h]: Will fluid_density be changed as well in another PR?

Comment (FZill) [reply ProcessLib/HydroMechanics/HydroMechanicsProcessData.h]: Yes, in the next PR. Shouldn't take too long.
